### ISSUE TITLE
volume_present is destroying volumes if `driver` is not specified

### ISSUE THREAD

**Issue by timcharper** [saltstack/salt#31997]
### Description of Issue/Question
In SaltStack `2015.8.7`, the volume is still being removed if driver isn't specified. This does not align with what the documentation advertises (in fact, providing `force: False` flag does nothing but generate a warning that said flag is an invalid keyword argument)

### Setup / Reproduce

```
[root@host ~]# salt-call state.single dockerng.volume_present derp
[INFO    ] Loading fresh modules for state activity
[INFO    ] Running state [derp] at time 17:07:59.116849
[INFO    ] Executing state dockerng.volume_present for derp
[INFO    ] {'removed': True, 'created': {u'Mountpoint': u'/var/lib/docker/volumes/derp/_data', u'Driver': u'local', u'Name': u'derp'}}
[INFO    ] Completed state [derp] at time 17:08:01.164826
local:
----------
          ID: derp
    Function: dockerng.volume_present
      Result: True
     Comment:
     Started: 17:07:59.116849
    Duration: 2047.977 ms
     Changes:
              ----------
              created:
                  ----------
                  Driver:
                      local
                  Mountpoint:
                      /var/lib/docker/volumes/derp/_data
                  Name:
                      derp
              removed:
                  True

Summary for local
------------
Succeeded: 1 (changed=1)
Failed:    0
------------
Total states run:     1
[root@host ~]# touch /var/lib/docker/volumes/derp/_data/hello
[root@host ~]# salt-call state.single dockerng.volume_present derp
[INFO    ] Loading fresh modules for state activity
[INFO    ] Running state [derp] at time 17:08:10.158343
[INFO    ] Executing state dockerng.volume_present for derp
[INFO    ] {'removed': True, 'created': {u'Mountpoint': u'/var/lib/docker/volumes/derp/_data', u'Driver': u'local', u'Name': u'derp'}}
[INFO    ] Completed state [derp] at time 17:08:10.166796
local:
----------
          ID: derp
    Function: dockerng.volume_present
      Result: True
     Comment:
     Started: 17:08:10.158343
    Duration: 8.453 ms
     Changes:
              ----------
              created:
                  ----------
                  Driver:
                      local
                  Mountpoint:
                      /var/lib/docker/volumes/derp/_data
                  Name:
                      derp
              removed:
                  True

Summary for local
------------
Succeeded: 1 (changed=1)
Failed:    0
------------
Total states run:     1
[root@host ~]# cat /var/lib/docker/volumes/derp/_data/hello
cat: /var/lib/docker/volumes/derp/_data/hello: No such file or directory
[root@host ~]#
```

### Versions Report
```
[root@host ~]# salt-call --versions-report
Salt Version:
           Salt: 2015.8.7

Dependency Versions:
         Jinja2: 2.7.2
       M2Crypto: 0.21.1
           Mako: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.7.0
         Python: 2.7.5 (default, Nov 20 2015, 02:00:19)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
        libgit2: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.7
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: Not Installed
        timelib: Not Installed

System Versions:
           dist: centos 7.2.1511 Core
        machine: x86_64
        release: 3.10.0-327.4.4.el7.x86_64
         system: CentOS Linux 7.2.1511 Core
```

**Comment by timcharper** [comment]
also relevant:

```
[root@host ~]# cat /usr/lib/python2.7/site-packages/docker/version.py
version = "1.6.0"
version_info = tuple([int(d) for d in version.split("-")[0].split(".")])
```

**Comment by timcharper** [comment]
I've confirmed that this is fixed in the latest head of `2015.8`, while not in `2015.8.7`, so I presume it will be fixed in `2015.8.8`. Closing.
